### ISSUE TITLE
Failed clone wiringPi

### ISSUE THREAD

**Issue by suisho** [nebrius/raspi-wiringpi#4]
Hi

I try install raspi-io package on raspberry pi, but got error

```
Installing I2C tools
Reading package lists... Done
Building dependency tree
Reading state information... Done
libi2c-dev is already the newest version.
0 upgraded, 0 newly installed, 0 to remove and 69 not upgraded.

Downloading Wiring Pi...

Cloning into 'wiringPi'...
fatal: read error: Connection reset by peer
Could not download Wiring Pi

npm ERR! Linux 3.18.7-v7+
npm ERR! argv "/usr/local/bin/node" "/usr/local/bin/npm" "i" "raspi-wiringpi"
npm ERR! node v0.12.1
npm ERR! npm  v2.5.1
npm ERR! code ELIFECYCLE

npm ERR! raspi-wiringpi@1.0.4 install: `sh install-wiringpi.sh`
npm ERR! Exit status 1
npm ERR!
npm ERR! Failed at the raspi-wiringpi@1.0.4 install script 'sh install-wiringpi.sh'.
npm ERR! This is most likely a problem with the raspi-wiringpi package,
npm ERR! not with npm itself.
npm ERR! Tell the author that this fails on your system:
npm ERR!     sh install-wiringpi.sh
npm ERR! You can get their info via:
npm ERR!     npm owner ls raspi-wiringpi
npm ERR! There is likely additional logging output above.
```

And I try clone wiringPi, can't this

```
% git clone git://git.drogon.net/wiringPi                       
Cloning into 'wiringPi'...
fatal: read error: Connection reset by peer
```

**Comment by bryan-m-hughes** [comment]
Looks like it's down for me too. I set up my own mirror of Wiring Pi on GitHub to get around this issue now and in the future.

Fixed in v1.0.5 of raspi-wiringpi and v3.1.1 of raspi-io

**Comment by suisho** [comment]
Thank you!
